### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mishakoshilka**
[DEMO LINK](https://mishakoshilka.github.io/layout_creativeBakery/).

### CODE REVIEW

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added comments to fix: 
Screenshot 2023-08-10 at 17 32 56
Screenshot 2023-08-10 at 17 34 10
The user should be able to call while clicking on this link
Screenshot 2023-08-10 at 17 34 44
Social media icons should follow to the new blank page of appropriate social media

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Nice work! Almost done: 
Screenshot 2023-08-10 at 18 31 45
Consider fixing validation errors. User should be able to call by clicking on the phone link in the header

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,311 @@
...
+    </header>
+
+    <aside class="page page__menu" id="menu">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo-link">
+          <img
+          src="images/Logo.png"
+          alt="Logo bakery"
+          class="top-bar__logo"
+        >
```
Comment (volodymyr-soltys97) [inline src/index.html:76]: Need to fix and check in everywhere
```suggestion
          <img
            src="images/Logo.png"
            alt="Logo bakery"
            class="top-bar__logo"
          >
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,311 @@
...
+            </a>
+            <a
+              href="https://www.instagram.com/"
+              class="footer__icon icon--instagram"
+              target="_blank">
+            </a>
+            <a
+              href="https://twitter.com/"
+              class="footer__icon icon--twitter"
+              target="_blank">
```
Comment (volodymyr-soltys97) [inline src/index.html:302]: Need to fix it everywhere
```suggestion
            <a
              href="https://twitter.com/"
              class="footer__icon icon--twitter"
              target="_blank"
            >
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Add favicon on the page
image

2. Add hover effects here
image

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**